futurex_openedx_extensions/dashboard/details/learners.py

```diff
@@ -154,18 +158,35 @@ def get_learners_queryset(
     :return: QuerySet of learners
     :rtype: QuerySet
     """
-    tenant_sites = get_tenants_sites(fx_permission_info['permitted_tenant_ids'])
+    full_access_tenant_ids = set()
```

Comment:
### Partial access logic into helpers
The new logic here looks like it should be added should probably be abstracted into shared helpers among other views.

This view also needs the `{ "limited_access": bool(partial_access) }` parameter in the returned JSON value.

### `fx_permission_info` to indicate partial access
I'm thinking that we should pass the `fx_permission_info` into all responses so the frontend app can decide what to do with them.

Reply:
Thank you @OmarIthawi , I've done the needed refactor in https://github.com/nelc/futurex-openedx-extensions/pull/80. The code in this PR looks cleaner now